config/default_records/instrument_processes_instruments/002_destroying_process_instrument.yml

```diff
@@ -0,0 +1,12 @@
+# Add entries to this file to generate them automatically upon running
+# rake record_loader:all or rake record_loader:<%= underscore %>
+# If you want entries which are only created in development mode then simply
+# create a file in this directory ending in '.dev.yml'
+# You can add as many additional .yml files to this directory as you wish.
+# Note: The keys here are for reference only and instrument process instruments have no name
+
+# Add a new mapping between Destroying instrument process and Destroying location instrument
+Destroying instrument - Destroying location:
+  instrument: Destroying instrument
+  instrument_process: destroy_location
+  bed_verification_type: Verification::DestroyLocation::Base
```

Comment:
I'm assuming this is here because it's mandatory to specify it? That's a little weird 😅

Reply:
Yes, it is mandatory